source/extensions/filters/listener/tls_inspector/tls_inspector.h

```diff
@@ -81,6 +82,22 @@ class Config {
 
 using ConfigSharedPtr = std::shared_ptr<Config>;
 
+/**
+ * TLS inspector filter state
+ */
+class FilterState : public Envoy::StreamInfo::FilterState::Object {
```

Comment:
Given that this is just some simple state, would it make more sense to make this dynamic metadata? That would be able to, for example, include in an access log.

Reply:
I don't have an opinion either way. Let me know if you prefer to use dynamic metadata and I can change to that.

Reply:
done